lib/paperclip/color_extractor.rb

```diff
@@ -26,8 +26,9 @@ def make
 
       foreground_palette.each do |color|
         distance = ColorDiff.between(background_color, color)
+        contrast = w3c_contrast(background_color, color)
 
-        if distance > max_distance
+        if distance > max_distance && contrast >= MIN_CONTRAST
```

Comment:
I omitted this on purpose though so we have a higher chance of getting a different color for the accent since the accent does not really have to be so contrasting, and I tested with a bunch of different album art to ensure it all looked nice.

Reply:
Alright, but we probably need at least *some* contrast. In my example of an album art with only two very-close colors, the accent would be indistinguishable from the background.

Reply:
That's fair enough, I suggest using a different contrast threshold then, something more relaxed.